### ISSUE TITLE
Don't hard cut contact name

### ISSUE THREAD

**Issue by Facyla** [linagora/twake-on-matrix#1462]
Web client on Ubuntu FF

The contact name is cut to 10 caracters (9 + "..."), which is not good for accessibility (partial information on a link/action button) + really short for all users. Also the full name is not accessible until one clicks on the contact.

Suggestions : 
1. Don't cut the name, and only use visual effects to avoid overflow and uncontrolled dimensions
2. Wrap the full text under the contact image so that one can read a little more than max 10 caracters, eg. max 2 lines with overflow:hidden so that the extra text remains accessible, but does not display when overflowing from eg. 2 lines.
3. Add the name as title attribute so it can display when hovering over the contact

![image](https://github.com/linagora/twake-on-matrix/assets/286492/73945d41-d516-4a7f-bc70-c6e8bfa53e21)
